.github/workflows/test.yml

```diff
@@ -64,46 +65,18 @@ jobs:
       # - phpcs_version: The PHPCS versions to test against.
       #   IMPORTANT: test runs shouldn't fail because of PHPCS being incompatible with a PHP version.
       #   - PHPCS will run without errors on PHP 5.4 - 7.4 on any supported version.
-      #   - PHP 8.0 needs PHPCS 3.5.7+ to run without errors.
+      #   - PHP 8.0 needs PHPCS 3.5.7+ to run without errors, and we require a higher minimum version.
+      #   - PHP 8.1 needs PHPCS 3.6.1+ to run without errors, but works best with 3.7.1+, and we require at least this minimum version.
```

Comment:
Do these comments still have value once the minimum supported PHPCS version has been raised to PHPCS 3.7.1 ?

Reply:
To me they do - they explain some of the decisions of why we can test with certain PHP versions and PHPCS versions.